saidump/saidump.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+extern "C" {
+#include <sai.h>
+}
+#include <inttypes.h>
+#include <set>
+#include <regex>
+#include <climits>
+#include <getopt.h>
+#include "swss/table.h"
+#include "meta/sai_serialize.h"
+#include "sairediscommon.h"
+#include <nlohmann/json.hpp>
```

Comment:
please remove all unnecessary includes that are not used in header file, and move them to cpp files, for example sairediscommon.h is not used in this header, but in cpp, same for regex and getopt and others

Reply:
Fixed.